buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -36,4 +37,6 @@ object Libs {
     const val FRAGMENT_KTX = "androidx.fragment:fragment-ktx:${Versions.FRAGMENT_KTX}"
     const val MOCKITO_INLINE = "org.mockito:mockito-inline:${Versions.MOCKITO}"
     const val LIFECYCLE = "androidx.lifecycle:lifecycle-common-java8:${Versions.LIFECYCLE}"
+    const val VIEW_BINDING_DELEGATE = "com.github.kirich1409:" +
+            "viewbindingpropertydelegate-noreflection:${Versions.VIEW_BINDING_DELEGATE}"
```

Comment:
Perhaps we can add a rule to allow those constants to bypass maxLineLength

Reply:
Yeah, we can suppress. It will make it more readable 👍